src/content/pages/programme/mentorship.mdx

```diff
@@ -42,12 +42,29 @@ with your mentee to:
 
 You will also be invited to join the panel of the Webinar Workshops.
 
+<div style={{textAlign: "center"}}>
+    <ButtonLink href="https://forms.gle/Wbn9Z7zBQtK56xjS9">I would like to share my experience by being a "Mentor".</ButtonLink>
+</div>
+
 ## Mentee
 
 If you require help contributing to EuroPython, especially if you are from an
 underrepresented or a marginalised group in the tech industry, you are welcome
 to submit your application once the form is opened and become a mentee.
 
+<div style={{textAlign: "center"}}>
+    <ButtonLink href="https://forms.gle/HgyoWnp3sc89S5DbA">I would like to receive help by being a "Mentee".</ButtonLink>
+</div>
+
 ## 2. Workshops
 
-More information coming soon.
+### Online Event: First Time Speaker’s Workshop
+After the CfP we will have an online panel session with 2-3 experienced community members sharing advice for the first-time (or other) speakers. At the end of the panel, there will be a QnA session to answer all the participants' questions.
```

Comment:
Could we decide whether we want to use "CfP" or "CFP" and then be consistent throughout? :D

Reply:
I adjusted according to  your suggestions.